### ISSUE TITLE
Feature request: Icon Elements to be inline

### ISSUE THREAD

**Issue by pingram3541** [pojome/elementor#1601]
So many times I have to resolve to using html or text element to get a series of icons w/ links horizontally aligned.  Would be great if there was a toggle to allow the icon element to act as an inline sibling vs block.  Thx.

**Comment by arielk** [comment]
Is it related to this issue #1624 ?

**Comment by pingram3541** [comment]
Yes similar in intent but if you see the title it is for a different element in different circumstances as all styling applies individually vs an entire list of icons.  For the single icon element I realize in edit mode it wouldn't make sense to have any element non-block but if one could check a box to toggle inline vs left, right, center and simply preview to see multiple icons that were stacked then shown inline.  One would set the preferred block alignment in that case via the column properties.  I hope this brings more clarity. 
 Thx for considering this =)
